app/helpers/mailer_helper.rb

```diff
@@ -182,7 +182,7 @@ def style_work_tag_metadata(tags)
 
   def commenter_pseud_or_name_link(comment)
     if comment.comment_owner.nil?
-      style_bold(comment.comment_owner_name) + style_role(t("roles.guest_with_parens"))
+      style_bold(comment.comment_owner_name) + " " + style_role(t("roles.guest_with_parens"))
```

Comment:
```suggestion
      "#{style_bold(comment.comment_owner_name)} #{style_role(t('roles.guest_with_parens'))}".html_safe
```
This would make rubocop happier. I didn't suggest it at first because calling `html_safe` is discouraged. But both style_link and style_bold already call it on their user provided input, so it should be fine.

I don't know which open is better here - making rubocop happy or avoiding the html_safe call.

Reply:
Would using safe_join work here? Something like: 

```
safe_join([style_bold(comment.comment_owner_name), ' ', style_role(t('roles.guest_with_parens'))])
```